postcss.config.js

```diff
@@ -2,6 +2,5 @@ module.exports = {
   plugins: {
     'postcss-import': {},
     autoprefixer: {},
-    cssnano: {},
```

Comment:
Removing this plugin because the `OptimizeCssAssetsPlugin` in Webpack replaces it